### ISSUE TITLE
FATA[3760] salt-call completed but had failed states

### ISSUE THREAD

**Issue by jofrep** [ekristen/cast#136]
Installing teamdfir/sift ends with errors.
OS is Ubuntu 20.04.6 LTS  and cast version is v0.14.52 (I created the folder /etc/apt/keyrings/)

```
INFO[3757] state completed                               component=installer duration=0.91 state=sift-config-tools time_begin="23:32:13.331764" time_end="23:32:13.332675"
INFO[3757] state completed                               component=installer duration=5.03 state=sift-config time_begin="23:32:13.343946" time_end="23:32:13.348975"
INFO[3757] state completed                               component=installer duration=0.726 state=sift-desktop-include time_begin="23:32:13.349175" time_end="23:32:13.349901"
INFO[3760] log file location                             component=installer file=/var/cache/cast/installer/logs/saltstack.log
INFO[3760] results file location                         component=installer file=/var/cache/cast/installer/logs/results.yaml
WARN[3760] first failed state                            comment="One or more requisite failed: sift.python-packages.volatility.sift-python-volatility-community-plugins" component=installer run_num=285 sls=sift.python-packages.volatility
INFO[3760] statistics                                    component=installer failed=43 success=630 total=673
INFO[3760] salt-call completed but had failed states     component=installer
FATA[3760] salt-call completed but had failed states
```

	
[saltstack.log](https://github.com/user-attachments/files/17604760/saltstack.log)

**Comment by ekristen** [comment]
Yup. Just discovered. Digging into that now

**Comment by ekristen** [comment]
@jofrep this is fixed with 2024.11.01 release of sift.
